galaxy_importer/ansible_test/job_template.yaml

```diff
@@ -14,11 +14,8 @@ spec:
       - name: {job_name}
         image: {image}
         resources:
-          requests:
```

Comment:
I don't think removing `requests:` is a good idea. If container is started with `limits`, but no `requests`, the `requests` will be (not 100% sure, but according to [1]) set to match limit, which is `1Gi`.

Also it would be nice to configure such job parameters, that environment dependent too as galaxy-importer parameters. So that it can be tuned for a specific environment without having to release a new version of a `galaxy-importer`.

[1] https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/

Reply:
added parameters with requests